app/auth/views.py

```diff
@@ -1,11 +1,21 @@
+from datetime import datetime
```

Comment:
is it time to split this out?  create a separate file for the signup/create user routes?